lib/ios/RNNModalManager.m

```diff
@@ -115,5 +118,19 @@ -(UIViewController*)topPresentedVCLeaf {
 	return [root topViewController] ? [root topViewController] : root;
 }
 
+- (UIViewController *)topUIViewController:(UIViewController *)viewController {
```

Comment:
Need to find the closest UIViewController because `viewController` is `RNNRootViewController` from non-stacked modals, but `RNNNavigationController` for stacked modals